src/components/Home/AuthorList.jsx

```diff
@@ -5,19 +5,19 @@ import AuthorListItem from './AuthorListItem';
 class AuthorList extends Component {
   renderListItem(item) {
     return (
-      <div>
+      <li key={item.id} className="list-group-item pl-0 pr-0">
         <AuthorListItem item={item} />
-      </div>
+      </li>
     );
   }
   render() {
     const items = this.props.items;
```

Comment:
a more specific name like 'author' can replace 'item'

Reply:
I would suggest use "items" for all the list component and "item" for all listItem.